instructions.json

```diff
@@ -18,25 +18,35 @@
             "defaultInstruction": "Head {modifier}[ on {way_name}]"
         },
         "end of road": {
-            "defaultInstruction": "Turn {modifier}[ onto {way_name}]"
+            "defaultInstruction": "Turn {modifier}[ onto {way_name}] at the end of the road",
```

Comment:
Wouldn't it be low priority (https://github.com/mapbox/directions/blob/master/user-guidance.md#step-terminology) which would replace the distance? In addition, the user would need to know about our preference not to add the distance in that case. After all, end of road provides all you need to know in terms of when to take the turn.